grassroots-backend/src/testing/typetests/cast.test.ts

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
So here's a fun puzzle...

Without this, the commit checks don't pass, due to the type errors in this file.

However, with this, the tests no longer pass!

Excluding the file from the tsconfig also causes the test to fail.

Any thoughts?

Reply:
Aha, got it.

So I've got separate tests for type assertions and functional testing, and a tsconfig.json specifically for the type assertion test.

The type assertion test is then excluded from the tsconfig.json that we actually typecheck. A bit annoying, but works fine!